packages/svelte/src/compiler/phases/3-transform/server/transform-server.js

```diff
@@ -624,12 +624,30 @@ const javascript_visitors_runes = {
 		}
 		context.next();
 	},
-	CallExpression(node, { state, next }) {
+	CallExpression(node, { state, next, visit }) {
 		const rune = get_rune(node, state.scope);
 
 		if (rune === '$effect.active') {
 			return b.literal(false);
 		}
+		if (rune === '$log' || rune === '$log.break' || rune === '$log.trace') {
```

Comment:
this actually raises an interesting question — do we want `$log.break` and `$log.trace` to do anything initially, before anything has changed? I suspect not, and that by extension they should be no-ops on the server

Reply:
Agreed. Feel free to change to that behavior

Reply:
Actually break will run on init. Maybe we should change that

Reply:
on it